tests/Console/Report/FixReport/TextReporterTest.php

```diff
@@ -104,7 +104,7 @@ protected function createComplexReport(): string
 <comment>      ----------- end diff -----------</comment>
 
 
-Checked all files in 1.234 seconds, 2.500 MB memory used
+Checked 10 files in 1.234 seconds, 2.500 MB memory used
```

Comment:
I'm thinking it would be nice to have the number of files that would be fixed. What about
```suggestion
Found 1 of 10 files that can be fixed in 1.234 seconds, 2.500 MB memory used
```